### PR TITLE
31553: Improve Snapshots Info on What Gets Backed Up and Limitations

### PR DESCRIPTION
**Pull request by jonquil2002**
SC: https://app.shortcut.com/replicated/story/31553/docs-better-document-the-detail-of-what-a-full-snapshot-backs-up-customer-feedback

Previews:

- https://deploy-preview-519--replicated-docs.netlify.app/vendor/snapshots-overview#what-data-is-backed-up

- https://deploy-preview-519--replicated-docs.netlify.app/enterprise/snapshots-understanding

### CODE REVIEW

File: docs/enterprise/snapshots-understanding.md [hunk 1]
```diff
@@ -1,19 +1,40 @@
...
+
+    Object-stored data that does not use PVCs, such as Rook and Ceph, require custom hooks for backups. Custom hooks are configured by your vendor. See [Full Snapshots](#full).
+
+  * **Partial snapshots**: Backs up the application volumes and manifest files. See [Partial Snapshots](#partial).
 
 Snapshots are useful for rollback and disaster recovery scenarios. They are not intended to be used for application migration scenarios.
 
+## Limitations and Considerations
+
+- By default, snapshots are stored locally on the cluster. For production environments, Replicated recommends storing snapshots externally to prevent losing snapshots during a complete cluster outage.
```
Comment (sgalsaleh) [inline docs/enterprise/snapshots-understanding.md:22]: > By default, snapshots are stored locally on the cluster

This is only true for embedded clusters. Existing clusters don't come pre-configured with a default internal store, the end customer can configure where snapshots are stored when installing Velero.

File: docs/vendor/snapshots-overview.md [hunk 2]
```diff
@@ -1,19 +1,32 @@
...
 
 :::note
 The restore process is managed through the kots CLI only.
 :::
 
 The app manager also exposes hooks that can be used to inject scripts to execute with snapshots both [before and after a backup](snapshots-configuring-backups) and [before and after a restore](../enterprise/snapshots-understanding).
 
+## What Data is Backed Up
+
+There are two types of snapshots available that backup different types of data:
```
Comment (sgalsaleh) [inline docs/vendor/snapshots-overview.md:19]:
```suggestion
There are two types of snapshots available that back up different types of data:
```

File: docs/enterprise/snapshots-understanding.md (hunk 1 above)
Comment (paigecalvert) [reply docs/enterprise/snapshots-understanding.md]: Hi @sgalsaleh, I was just taking a look at this while Jonquil is out this week. Thanks very much for your review. I updated this bullet point based on your comment--do the changes look alright to you? Anything else that would need to be updated from your perspective before we merge this in?

Comment (jonquil2002) [reply docs/enterprise/snapshots-understanding.md]: @sgalsaleh Can you take a look at Paige's changes based on your comments, and also see whether anything else needs changing before we merge? Thanks.

**Pull request merged**